packages/lit-dev-server/src/middleware/content-security-policy-middleware.ts

```diff
@@ -0,0 +1,120 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import type Koa from 'koa';
+
+/**
+ * Options for contentSecurityPolicyMiddleware.
+ */
+export interface ContentSecurityPolicyMiddlewareOptions {
+  /**
+   * Enables some additional directives that are only required when running in
+   * local dev mode.
+   */
+  devMode?: boolean;
```

Comment:
I see! So directives are like a very specific allowlist. In devMode we allow `http` for example.

Reply:
Yep! https://web.dev/strict-csp/ is a great article.